src/gui/canvas.cpp

```diff
@@ -216,3 +217,9 @@ void canvas::invoke_delete()
 {
 	cm->activate_command(cm->find_command("incmdDeleteShape"));
 }
+void canvas::abordCommand()
```

Comment:
we have abort command, and it also generates log.
dicmdAbortCommand().execute()